### PR TITLE
V5 ga 89 mobile sdk editor

### PR DESCRIPTION
**Pull request by jiggsaw85**
**Jira task:** [https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?label=Screen_Conversion&selectedIssue=V5GA-89](https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?label=Screen_Conversion&selectedIssue=V5GA-89)

### CODE REVIEW

File: src/assets/styles/main.scss [hunk 1]
```diff
@@ -543,3 +543,23 @@ $color-light: hsl(0, 0%, 96%);
...
+  width: 125px;
+}
+
+.w-140 {
+  width: 140px;
+}
+
+.w-160 {
+  width: 260px;
+}
```
Comment (Aviv-Galmidi) [inline src/assets/styles/main.scss:557]: Please keep our style for width classes and place them at the top of the file with the rest of the width classes (in order of narrow(up)->wide(down)
```
.width-70px {
  max-width: 70px;
  min-width: 70px;
  width: 70px;
}
```

**Review by Aviv-Galmidi — COMMENTED** [review]
1. Getting errors for focus, missing ref?

Screen Shot 2023-05-10 at 12 03 46
Screen Shot 2023-05-10 at 12 05 11

2. Console is full of prop warnings when opening the page, please take a look at them

Screen Shot 2023-05-10 at 12 06 01

3. Adding signature doesn't work


https://github.com/reblaze/ui/assets/10487148/331ac7d1-f70e-400a-9451-ee888ca086b0

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
1. Getting errors for focus, missing ref?

Screen Shot 2023-05-10 at 12 03 46
Screen Shot 2023-05-10 at 12 05 11

2. Console is full of prop warnings when opening the page, please take a look at them

Screen Shot 2023-05-10 at 12 06 01

3. Adding signature doesn't work


https://github.com/reblaze/ui/assets/10487148/331ac7d1-f70e-400a-9451-ee888ca086b0

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
1. Getting errors for focus, missing ref?

Screen Shot 2023-05-10 at 12 03 46
Screen Shot 2023-05-10 at 12 05 11

2. Console is full of prop warnings when opening the page, please take a look at them

Screen Shot 2023-05-10 at 12 06 01

3. Adding signature doesn't work


https://github.com/reblaze/ui/assets/10487148/331ac7d1-f70e-400a-9451-ee888ca086b0

**Comment by Aviv-Galmidi** [comment]
Awaiting PR fixes

**Comment by SrdjanReblaze** [comment]
@Aviv-Galmidi 

all issues fixed

**Pull request merged**